docs/cookbook/syncing/rpc.md

```diff
@@ -18,6 +18,9 @@ bitcoin-cli --chain=regtest getnewaddress
 
 # Mine 101 blocks
```

Comment:
This comment is outdated and needs updating

Reply:
Isn't this referring to the next line showing the usage of `generatetoaddress` as a way to mine 101 blocks?

Reply:
Oh yes you're right. Never see diffs early morning without caffeine